plugins/modules/packaging/language/ansible_galaxy_install.py

```diff
@@ -70,6 +70,17 @@
     description:
       - Acknowledge using Ansible 2.9 with its limitations, and prevents the module from generating warnings about them.
       - This option is completely ignored if using a version of Ansible greater than C(2.9.x).
+      - Note that this option will be removed without any further deprecation warning once support
+        for Ansible 2.9 is removed from this module.
+    type: bool
+    default: false
+  ack_min_ansiblecore211:
+    description:
+      - Acknowledge the module is deprecating support for Ansible 2.9 and Ansible Core 2.10.
+      - Support for those versions will be removed in community.general 8.0.0.
+        At the same time, this option will be removed without any deprecation warning!
+      - This option is completely ignored if using a version of Ansible greater than C(2.9.x).
```

Comment:
```suggestion
      - This option is completely ignored if using a version of ansible-core/ansible-base/Ansible greater than C(2.9.x).
```
But: shouldn't this be ansible-core >= 2.11?

Reply:
```suggestion
      - This option is completely ignored if using a version of ansible-core/ansible-base/Ansible greater than C(2.11).
```

Reply:
Indeed it should.